src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Get/GetDialogDto.cs

```diff
@@ -12,183 +12,644 @@ namespace Digdir.Domain.Dialogporten.Application.Features.V1.EndUser.Dialogs.Que
+
+    /// <summary>
+    /// Arbitrary string indicating the version of the endpoint.
+    ///
+    /// Consult the service-specific documentation provided by the service owner for details (if in use).
+    /// </summary>
     public string? Version { get; set; }
+
+    /// <summary>
+    /// The fully qualified HTTPS URL of the API endpoint. Will be set to "urn:dialogporten:unauthorized" if the user is
+    /// not authorized to perform the action.
+    /// </summary>
+    /// <example>
+    /// https://someendpoint.com/api/v1/someaction
+    /// urn:dialogporten:unauthorized
+    /// </example>
     public Uri Url { get; set; } = null!;
+
+    /// <summary>
+    /// The HTTP method that the endpoint expects for this action.
+    /// </summary>
     public HttpVerb.Values HttpMethod { get; set; }
+
+    /// <summary>
+    /// Link to service provider documentation for the endpoint. Used for service owners to provide documentation for
+    /// integrators. Should be a URL to a human-readable page.
+    /// </summary>
     public Uri? DocumentationUrl { get; set; }
+
+    /// <summary>
+    /// Link to the request schema for the endpoint. Used by service owners to provide documentation for integrators.
+    /// Dialogporten will not validate information on this endpoint.
+    /// </summary>
     public Uri? RequestSchema { get; set; }
+
+    /// <summary>
+    /// Link to the response schema for the endpoint. Used for service owners to provide documentation for integrators.
+    /// Dialogporten will not validate information on this endpoint.
+    /// </summary>
     public Uri? ResponseSchema { get; set; }
+
+    /// <summary>
+    /// Boolean indicating if the endpoint is deprecated. Integrators should migrate to endpoints with a higher version.
+    /// </summary>
     public bool Deprecated { get; set; }
+
+    /// <summary>
+    /// Date and time when the service owner has indicated that endpoint will no longer function. Only set if the endpoint
+    /// is deprecated. Dialogporten will not enforce this date.
+    /// </summary>
     public DateTimeOffset? SunsetAt { get; set; }
 }
 
 public sealed class GetDialogDialogGuiActionDto
 {
+    /// <summary>
+    /// The unique identifier for the action in UUIDv7 format.
+    /// </summary>
     public Guid Id { get; set; }
+
+    /// <summary>
+    /// The action identifier for the action, corresponding to the "action" attributeId used in the XACML service policy,
+    /// </summary>
     public string Action { get; set; } = null!;
+
+    /// <summary>
+    /// The fully qualified HTTPS URL of the action, to which the user will be redirected when the action is triggered. Will be set to
+    /// "urn:dialogporten:unauthorized" if the user is not authorized to perform the action.
+    /// </summary>
+    /// <example>
+    /// urn:dialogporten:unauthorized
+    /// https://someendpoint.com/gui/some-service-instance-id
+    /// </example>
     public Uri Url { get; set; } = null!;
```

Comment:
Vi har ikke noe validering av det jeg kan se i dialogporten i dag som verifiserer at det tjenesteeiere sender inn her faktisk er HTTPS. Er det noe vi burde lage en oppgave på?

Reply:
Right, jeg blandet med sjekken på anchor-tags i HTML. Jeg tror kanskje ikke vi skal ha noen validering her, kanskje man på et eller annet tidspunkt ønsker å kunne bruke et annen schema for å trigge en app eller whatever. Og under testing/utvikling vil en gjerne kunne bruke bare HTTP. Skal endre litt på formuleringen.